app/graphql/types/fees/amount_details/graduated_percentage_range.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Types
+  module Fees
+    module AmountDetails
+      class GraduatedPercentageRange < Types::BaseObject
+        graphql_name 'FeeAmountDetailsGraduatedPercentageRange'
+
+        field :units, String, null: true
+        field :from_value, Integer, null: true
+        field :to_value, Integer, null: true
+        field :flat_unit_amount, String, null: true
+        field :rate, String, null: true
+        field :per_unit_total_amount, String, null: true
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `per_unit_total_amount` should appear before `rate`.